visualization_and_plotting/readme.md

```diff
@@ -0,0 +1,43 @@
+#Simple horizontal Bar Plots
+---
+#R
+```r
+library(dplyr)
+library(magrittr)
+library(ggplot2)
+
+iris %>%
+    dplyr::group_by(Species) %>%
+    dplyr::summarise(Sepal.Length = mean(Sepal.Length)) %>%
+    ggplot(aes(Species, Sepal.Length)) + geom_bar(stat = 'identity') + 
```

Comment:
Hah, ok sounds good :)

you can take the boy out of Avant but you can't take ....:) 
@robertzk see what you've done :)